octokit/repositories_test.go

```diff
@@ -97,9 +98,11 @@ func TestRepositoresService_Create(t *testing.T) {
 	assert.Equal(t, "This is your first repo", repo.Description)
 	assert.False(t, repo.Private)
 	assert.True(t, repo.Fork)
-	assert.Equal(t, "https://api.github.com/repos/octocat/Hello-World", repo.URL)
+	assert.Equal(t, "https://api.github.com/repos/octocat/Hello-World",
```

Comment:
Based on testify's recent changes, I think we need `EqualValues` when comparing a string to a `Hyperlink`, right?